bundles/net.tourbook.common/src/net/tourbook/common/time/TimeTools.java

```diff
@@ -232,10 +231,6 @@
    }
 
    public static String constructLocalExpireAtDateTime(final long expireAt) {
```

Comment:
`createISODateTime` as method name ?

TimeTools.java contains common methods, so `constructLocalExpireAtDateTime` is a little bit specific

Reply:
Good idea, I'll do that in my next PR

Reply:
Renamed to `getUTCISODateTime()`